atomizer.config.js

```diff
@@ -7,11 +7,19 @@ module.exports = {
     },
     content: ['app/components/', 'app/assets/js', 'docs/'],
     custom: {
-        borderTopLight: '1px solid rgba(2, 128, 174, 0.3)',
+        borderTopLight: '1px solid var(--color-border-light)',
         boxColorLight: 'rgba(2, 128, 174, 0.05)',
         boxColorDarker: 'rgba(2, 128, 174, 0.2)',
-        boxShadowLight: '0 2px 3px rgba(0, 0, 0, 0.2)',
-        brandColor: '#0280ae',
+        hamburgerImage: 'url(https://s.yimg.com/os/acss/images/menu.d6610eea.png)',
```

Comment:
do we need darkmode burger icon? maybe we should move to SVG then can just use the color var on fill...

Reply:
👍 yup, I want to move to svg and animate to close icon